src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -33,14 +33,6 @@
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <OutputType>Exe</OutputType>
   </PropertyGroup>
-
-  <!--
-     In IDE scenario for full CLR projects default test platform is v1, which expects full pdbs for source information.
-     This can be removed once TPv2 is default for full CLR. Related issue https://github.com/Microsoft/vstest/issues/373.
-   -->
-  <PropertyGroup>
-    <DebugType Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">Full</DebugType>
```

Comment:
Removing this before fixing https://github.com/Microsoft/vstest/issues/800 breaks code coverage for .NET Full framework. We can change the condition to add Full debug type if user not specified.

Reply:
@smadala This is not the right place to change DebugType value. As I pointed out by the time this is evaluated its value is already used to calculate other variables.

Reply:
Starting next sprint, we will begin the work to add code coverage support for .NET Core in the presence of portable and embedded PDBs.

@tmat, in the meantime we will merge in this PR.